rllib/core/rl_module/tf/tests/test_tf_rl_module.py

```diff
@@ -95,6 +95,29 @@ def test_get_set_state(self):
         state2_after = module2.get_state()
         check(state, state2_after)
 
+    def test_serialize_deserialize(self):
```

Comment:
add a unit test that tests throwing an error for a non serializable construction of a module?

Reply:
done